util/logging/retrieve-cluster-logs.py

```diff
@@ -42,6 +42,7 @@ def err_and_exit(message):
 
 def get_log_group(args):
     """Get log group for args.cluster."""
+    # ToDo change log group name to include timestamp
     log_group_name = "/aws/parallelcluster/{}".format(args.cluster)
```

Comment:
We can retrieve the name from the output of the stack, we should add this output if not there.